packages/waku/src/lib/rsc/worker-impl.ts

```diff
@@ -3,7 +3,11 @@
 import url from 'node:url';
 import { parentPort } from 'node:worker_threads';
 import { Server } from 'node:http';
-import { createServer as viteCreateServer } from 'vite';
+import {
+  createServer as viteCreateServer,
+  resolveConfig as resolveViteConfig,
+  mergeConfig as mergeViteConfig,
```

Comment:
Let's make our naming convention consistent.
Do you prefer `resolveViteConfig` to `viteResolveConfig`?

Reply:
The first one is better I guess.

Reply:
Fine then. We should change everything. If it's small, it can be done in this PR.

Reply:
Please check all `import { doSomethign as doViteSomething} from 'vite'` throughout the code base. My previous code did `viteDoSomething`.